zerver/forms.py

```diff
@@ -312,3 +319,14 @@ def clean_emails(self) -> List[Text]:
             raise forms.ValidationError(_("Please enter at most 10 emails."))
 
         return emails
+
+def validate_recaptcha(response_token: str) -> None:
+    if settings.RECAPTCHA_SITEKEY:
```

Comment:
This condition is redundant -- we don't need it both here and at the caller. I think I like it better at the caller, so let's cut it here.

Reply:
Yep. Will do.